src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -433,17 +450,15 @@ private fun getLobbyDisplayPair(): List<ScoreboardElementType> {
 }
 
 private fun getPowerDisplayPair() = listOf(
-    when (MaxwellAPI.currentPower) {
-        null -> "§cOpen \"Your Bags\"!"
-        else ->
-            if (displayConfig.displayNumbersFirst) {
-                "§a${MaxwellAPI.currentPower?.replace("Power", "")} Power " +
-                    "§7(§6${MaxwellAPI.magicalPower}§7)"
-            } else {
-                "Power: §a${MaxwellAPI.currentPower?.replace("Power", "")} " +
-                    "§7(§6${MaxwellAPI.magicalPower?.addSeparators()}§7)"
-            }
-    } to HorizontalAlignment.LEFT
+    (MaxwellAPI.currentPower?.let {
+        val mp = if (displayConfig.showMagicalPower) "§7(§6${MaxwellAPI.magicalPower?.addSeparators()}§7)" else ""
+        if (displayConfig.displayNumbersFirst) {
+            "§a${MaxwellAPI.currentPower?.replace("Power", "")} Power $mp"
+        } else {
+            "Power: §a${MaxwellAPI.currentPower?.replace("Power", "")} $mp"
```

Comment:
use `it` and extract name.
```suggestion
        val name = it.replace("Power", "")
        if (displayConfig.displayNumbersFirst) {
            "§a$name Power $mp"
        } else {
            "Power: §a$name $mp"
```